File: SW Expert Academy/2058.c
#include <stdio.h>
#pragma warning (disable:4996)

main()
{
	int receive_number, result;

	printf("Enter a number :");
	scanf("%d", &receive_number);

	result = (receive_number / 1000) + (receive_number % 1000) / 100 + (receive_number % 100) / 10 + receive_number % 10;

	printf("%d\n", result);
}

File: 2021년/SW Expert Academy/Python/2058.py
# 하나의 자연수를 입력받아 각 자릿수의 합을 계산하라

receive_number = int(input())

result = (receive_number // 1000) + ((receive_number % 1000) // 100) + ((receive_number % 100) // 10) + receive_number % 10   
print(result)
